kubernetes/helm-charts/buildfarm/templates/shard-worker/statefulsets.yaml

```diff
@@ -70,8 +72,8 @@ spec:
             - mountPath: /config
               name: config
               readOnly: true
-            - mountPath: /tmp/worker
```

Comment:
Was this removal intentional?

Reply:
@werkt No (Sorry for not checking). I reverted the modification.